test/rules/ordered-imports/test.ts.lint

```diff
@@ -0,0 +1,22 @@
+// Named imports should be alphabetized.
+import {A, B} from 'foo';
+import {B, A} from 'foo'; // failure
+        ~~~~                         [ordered-imports]
+
+// Lowercase comes before uppercase.
+import {b, A, C} from 'foo';
+import {A, b, C} from 'foo'; // failure
+        ~~~~                            [ordered-imports]
+
+
+// Import sources should be alphabetized.
+import * as bar from 'bar';
+import * as foo from 'foo';
+
+import * as abc from 'abc';
+import * as foo from 'foo';
+import * as bar from 'bar'; // failure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~            [ordered-sources]
+
```

Comment:
Could we add a test case with this kind of import:
```
import someDefault from "module";
```

as well as the combined form:

```
import someDefault, {nameA, nameB: differentIdentifier} from "./wherever";
```

Just is a nice sanity check that the rule doesn't throw errors or struggle with all the different forms of imports